python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -11,7 +11,10 @@
 from dask.utils_test import hlg_layer
 
 import cudf
-from cudf.core._compat import PANDAS_CURRENT_SUPPORTED_VERSION, PANDAS_VERSION
+from cudf.core._compat import (
+    PANDAS_CURRENT_SUPPORTED_VERSION,
+    PANDAS_VERSION,
+)
```

Comment:
Was this done by pre-commit? I suppose if this is undone you won't need the dask codeowner review anymore.

Reply:
Yeah, somehow pre-commit keeps doing this. I'll see if I can revert this file and re-commit if CI complains.